### PR TITLE
New color scheme

### PR DESCRIPTION
**Pull request by pcowgill**
### Issue link

https://github.com/tasitlabs/tasit/issues/78

##### Other (please describe below if you select "Other")
UI styling

### Notes

@marcelomorgado I'm stuck getting the blockchain eye links to appear locally (which normally they do, so I'm trying to debug whatever started happening recently) so that I can confirm they're the right color

### CODE REVIEW

**Comment by pcowgill** [comment]
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 11 41 50
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 11 23 45
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 11 15 21
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 11 07 29
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 11 07 24
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 10 53 03
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 10 52 59
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 10 52 55
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 10 52 50
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 10 52 46
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 10 52 43 png
Simulator Screen Shot - iPhone Xʀ - 2019-05-08 at 17 56 13

**Comment by marcelomorgado** [comment]
Nice color scheme! Some lint errors.

**Comment by pcowgill** [comment]
After the keyboard color theme fix for iOS

Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 13 17 20

**Comment by pcowgill** [comment]
@marcelomorgado and I just talked on Slack, and we figured out that it was because it was using `0.0.18` of the SDK but without making the changes needed for the tx to go through.

I just tested the app on this branch running with SDK `0.0.17` and I see the `eye` icons as the right color now

Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 13 28 58

**Comment by pcowgill** [comment]
Here are the remaining screens I couldn't see while debugging the tx issue. I think after fixing the lint issues this can be merged.

Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 13 32 47
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 13 32 36
Simulator Screen Shot - iPhone Xʀ - 2019-05-09 at 13 32 33

**Comment by pcowgill** [comment]
@marcelomorgado For this lint error,

```
/Users/paulcowgill/Code/tasitlabs/tasit/decentraland/components/presentational/__snapshots__/EthereumSignUpForm.test.js.snap
  3:1  error  Expected Jest snapshot to be smaller than 50 lines but was 54 lines long  jest/no-large-snapshots
```

do you think the move is to strip the input field or the continue button into a react fragment that is then used in the parent component?

**Comment by marcelomorgado** [comment]
> @marcelomorgado For this lint error,
> 
> ```
> /Users/paulcowgill/Code/tasitlabs/tasit/decentraland/components/presentational/__snapshots__/EthereumSignUpForm.test.js.snap
>   3:1  error  Expected Jest snapshot to be smaller than 50 lines but was 54 lines long  jest/no-large-snapshots
> ```
> 
> do you think the best move is to strip the text input field or the continue button into a react fragment that is then used in the parent component?

Strip the continue button couldn't be enough to short the snapshot, so I think that strip the text input is the best move.

File: decentraland/components/presentational/EthereumSignUpForm.js [hunk 1]
```diff
@@ -44,6 +39,27 @@ export default class EthereumSignUpForm extends React.Component {
   }
 }
 
+export function TextInputField({ handleUsername, username }) {
```
Comment (marcelomorgado) [inline decentraland/components/presentational/EthereumSignUpForm.js]: Maybe a more specific name like `UsernameTextInput` or `UsernameInputField`?

Comment (pcowgill) [reply decentraland/components/presentational/EthereumSignUpForm.js]: Sure, good call

Comment (marcelomorgado) [inline decentraland/components/presentational/EthereumSignUpForm.js]: And maybe `onChange` instead of `handleUsername`?

Comment (pcowgill) [reply decentraland/components/presentational/EthereumSignUpForm.js]: I made this change - resolving this

Comment (pcowgill) [reply decentraland/components/presentational/EthereumSignUpForm.js]: Made this change too

**Comment by pcowgill** [comment]
Good to merge?

**Comment by marcelomorgado** [comment]
> Good to merge?

For sure.

**Pull request merged**